mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompilerConfig.java

```diff
@@ -32,6 +32,11 @@
         return map;
     }
 
+    /**
+     * Returns true if user didn't modify symbol prefixes
+     */
+    boolean isDefaultSymbolPrefixes();
```

Comment:
More of just a curiosity, but I've always meant to ask about the `symbolPrfixes` in the `PresentationCompilerConfig` interface. I know I don't actually expose this in `coc-metals`, and it looks like we don't in the VS Code extension either. Is this intentional or just an oversight that you originally never used this in the VS Code extension?

Reply:
I don't think this feature was used too much at any point, it has been implemented early with the other completion features by @olafurpg , but not sure what was the intention.